pkgs/tools/audio/liquidsoap/full.nix

```diff
@@ -98,8 +99,9 @@ stdenv.mkDerivation {
     ocamlPackages.fdkaac
     ocamlPackages.flac
     ocamlPackages.frei0r
-    ocamlPackages.gd4o
+    ocamlPackages.gd
     ocamlPackages.graphics
+    # ocamlPackages.gstreamer
```

Comment:
Remove?

Reply:
I was confused as to why gstreamer was not detected, realized it was just not included later. Added it commented out as it is broken, but as a reminder for why it isn't part of the build.

This is a big enough missing feature that I think spelling it out is warranted in this case.

Reply:
Can you add a comment about why it's commented out since it's confusing for someone like me?

Reply:
```suggestion
    # ocamlPackages.gstreamer # advertised feature, currently not detected by build
```